src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -353,56 +318,160 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
     var gLDebug = false; // true to support GLInspector addon
     var debugMode = false;
 
-    var ellipsoid = new Ellipsoid({
+    this.ellipsoid = new Ellipsoid({
         x: 6378137,
         y: 6356752.3142451793,
         z: 6378137,
     });
 
     var coordinate = new GeoCoordinate().copy(coordCarto, UNIT.DEGREE);
+    // TODO: use GeoCoordinate conversion instead
+    var positionCamera = this.ellipsoid.cartographicToCartesian(coordinate);
+    this.scene = new Scene(viewerDiv, debugMode, gLDebug);
+    this.scene.camera.setPosition(positionCamera);
 
-    this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
-
-    var map = new Globe(ellipsoid, gLDebug);
-
-    this.scene.add(map);
-
-    // Register all providers
-    var wmtsProvider = new WMTS_Provider({
-        support: map.gLDebug,
-    });
-
-    this.scene.scheduler.addProtocolProvider('wmts', wmtsProvider);
-    this.scene.scheduler.addProtocolProvider('wmtsc', wmtsProvider);
-    this.scene.scheduler.addProtocolProvider('tile', new TileProvider(ellipsoid));
-    this.scene.scheduler.addProtocolProvider('wms', new WMS_Provider({ support: map.gLDebug }));
+    this.scene.initDefaultProviders(this.scene);
 
     this.sceneLoadedDeferred = defer();
     this.addEventListener('globe-loaded', () => {
         this.sceneLoadedDeferred.resolve();
         this.sceneLoadedDeferred = defer();
     });
 
-    return this.scene;
+    // init globe layer with default parameter
+    const wgs84TileLayer = {
+        // layer base options
+        protocol: 'tile',
+        id: globeLayerId,
+        preUpdate: (context, layer) => preGlobeUpdate(context, layer),
```

Comment:
`preGlobeUpdate` Can be named `preUpdateTreeLayer`
or `preGlobeUpdate` is specific toh the Globe?